speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/unitTestUtilitiesTests/UtilsTests.cpp

```diff
@@ -142,4 +142,49 @@ TEST(UtilsTests, PairWiseCoverageTest)
             }
         }
     }
+}
+
+void PairWise2(bool p_1, bool p_2) {SUCCEED() << p_1 << " " << p_2;}
+void PairWise3(bool p_1, bool p_2, bool p_3) {SUCCEED() << p_1 << " " << p_2 << " " << p_3;}
+void PairWise4(bool p_1, bool p_2, bool p_3, bool p_4) {SUCCEED() << p_1 << " " << p_2 << " " << p_3 << " " << p_4;}
+void PairWise5(bool p_1, bool p_2, bool p_3, bool p_4, bool p_5) {SUCCEED() << p_1 << " " << p_2 << " " << p_3 << " " << p_4 << " " << p_5;}
+void PairWise6(bool p_1, bool p_2, bool p_3, bool p_4, bool p_5, bool p_6) {SUCCEED() << p_1 << " " << p_2 << " " << p_3 << " " << p_4 << " " << p_5 << " " << p_6;}
+void PairWise7(bool p_1, bool p_2, bool p_3, bool p_4, bool p_5, bool p_6, bool p_7) {SUCCEED() << p_1 << " " << p_2 << " " << p_3 << " " << p_4 << " " << p_5 << " " << p_6 << " " << p_7;}
+void PairWise8(bool p_1, bool p_2, bool p_3, bool p_4, bool p_5, bool p_6, bool p_7, bool p_8) {SUCCEED() << p_1 << " " << p_2 << " " << p_3 << " " << p_4 << " " << p_5 << " " << p_6 << " " << p_7 << " " << p_8;}
```

Comment:
No need to add documentation for all of them, but at least add it to the first one